runner/local/simple.go

```diff
@@ -198,7 +197,24 @@ func (r *simpleRunner) run(cmd *runner.Command, runId runner.RunId, doneCh chan
 
 	switch st.State {
 	case execer.COMPLETE:
-		r.updateStatus(runner.CompleteStatus(runId, st.ExitCode))
+		snapshotId := ""
+		srcToDest := map[string]string{
```

Comment:
We can address this in a new PR, and it's not super clean to require two copies of a file to ingest.